F1/F1/Utilities/ViewController+Extension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ViewController+Extension.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/05/09.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController{
```

Comment:
```suggestion
extension UIViewController {
```

Reply:
swiftlint should have picked this us. Please do a pr for going through all the linting errors